static/js/tools/scatter/chart.tsx

```diff
@@ -165,25 +165,22 @@ function Chart(props: ChartPropsType): JSX.Element {
     }
   }
 
-  // Replot when data changes.
+  // Replot when props or chart width changes.
   useEffect(() => {
+    const entrySet = new Set();
     if (props.display.chartType === ScatterChartType.MAP && !geoJsonFetched) {
       loadSpinner(CONTAINER_ID);
       return;
     } else {
       removeSpinner(CONTAINER_ID);
+      replot();
     }
-    replot();
-  }, [props, geoJsonFetched]);
-
-  // Replot when chart width changes on sv widget toggle.
-  useEffect(() => {
-    const debouncedHandler = _.debounce(() => {
-      if (
-        props.display.chartType === ScatterChartType.SCATTER ||
-        geoJsonFetched
-      ) {
+    const debouncedHandler = _.debounce((entries) => {
```

Comment:
could you add documentation for entries? i could not find reference to it :-/

Reply:
done